ee/maintained-apps/outputs/apps.json

```diff
@@ -230,7 +230,7 @@
       "slug": "yubico-yubikey-manager/darwin",
       "platform": "darwin",
       "unique_identifier": "com.yubico.ykman",
-      "description": "Requires Rosetta 2. YubiKey Manager is an application for configuring any YubiKey."
+      "description": "YubiKey Manager is an application for configuring any YubiKey. Requires Rosetta 2. YubiKey Manager won't get security updates or bug fixes. It's End of Life: https://www.yubico.com/support/download/yubikey-manager/"
```

Comment:
@eashaw will this URL render nicely on the website?

<img width="680" height="202" alt="Screenshot 2025-09-16 at 12 06 42 PM" src="https://github.com/user-attachments/assets/d29de6a3-bfc2-4f9b-86af-b7c86c209336" />

Reply:
Thanks!